<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QWidget" name="ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QPushButton" name="helpButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;What is this?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="configSel"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QStackedWidget" name="configStack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="NoCWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>No configuration loaded</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="XYZConfig" name="XYZWidget"/>
     <widget class="PWConfig" name="PWWidget"/>
     <widget class="LmpConfig" name="LmpWidget"/>
     <widget class="CPConfig" name="CPWidget"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PWConfig</class>
   <extends>QWidget</extends>
   <header>configs/pwconfig.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LmpConfig</class>
   <extends>QWidget</extends>
   <header>configs/lmpconfig.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XYZConfig</class>
   <extends>QWidget</extends>
   <header>configs/xyzconfig.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CPConfig</class>
   <extends>QWidget</extends>
   <header>configs/cpconfig.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
